coderdojochi/settings.py

```diff
@@ -25,7 +26,7 @@
 SECRET_KEY = 'e^u3u$pukt$s=6#&9oi9&jj5ow6563fuka%y9t7i*2laalk^l$'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = str_to_bool(os.environ.get('DEBUG')) or False
+DEBUG = str_to_bool(os.environ.get('DEBUG')) and 'test' not in sys.argv or False
```

Comment:
@ash6851 ah, yes. This needs to be set to default to false. I'll commit a fix ASAP.